src/odbc.cpp

```diff
@@ -348,37 +363,37 @@ class ConnectAsyncWorker : public ODBCAsyncWorker {
       // get information about the connection
       // maximum column length
       DEBUG_PRINTF("[SQLHENV: %p][SQLHDBC: %p] ODBC::ConnectAsyncWorker::Execute(): Calling SQLGetInfo(ConnectionHandle = %p, InfoType = %d (SQL_MAX_COLUMN_NAME_LEN), InfoValuePtr = %p, BufferLength = %lu, StringLengthPtr = %lu)\n", hEnv, hDBC, hDBC, SQL_MAX_COLUMN_NAME_LEN, &maxColumnNameLength, sizeof(SQLSMALLINT), NULL);
-      sqlReturnCode = SQLGetInfo(
+      return_code = SQLGetInfo(
         hDBC,                    // ConnectionHandle
```

Comment:
Looks like an hDBC -> hdbc change missed.

Reply:
Definitely looks like it, though merging in the removal of all the DEBUG_PRINTF so it will be moot.